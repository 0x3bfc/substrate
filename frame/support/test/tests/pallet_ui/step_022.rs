#[frame_support::pallet(Example)]
mod pallet {
	use frame_support::pallet_prelude::*;

	#[pallet::trait_]
	pub trait Trait: frame_system::Trait {}

	#[pallet::module]
	pub struct Module<T> {}

	#[pallet::module_interface]
	impl<T: Trait> ModuleInterface for Module<T> {}

	#[pallet::call]
	impl<T: Trait> Foo for Module<T> {}
}

fn main() {
}
