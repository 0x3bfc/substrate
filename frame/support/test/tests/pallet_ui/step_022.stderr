error: Invalid pallet::call, expect no trait ident as in `impl<..> Pallet<..> { .. }`
  --> $DIR/step_022.rs:15:22
   |
15 |     impl<T: Config> Foo for Pallet<T> {}
   |                         ^^^
